package.json

```diff
@@ -15,13 +15,13 @@
     "update-actions-build": "node actions/build.js"
   },
   "dependencies": {
-    "@actions/core": "^1.2.7",
+    "@actions/core": "^1.10.0",
     "@actions/github": "^2.2.0",
-    "axios": "^0.21.1",
-    "dotenv": "^9.0.2",
```

Comment:
This version change seems to be a big jump. Were u able to check the change logs if any functions we use have been changed?

Reply:
@gp201 Yes, most of the breaking changes are mostly just things like adding multiline parsing support: https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md